Rubberduck.RegexAssistant/Atom.cs

```diff
@@ -0,0 +1,284 @@
+﻿using Rubberduck.RegexAssistant.i18n;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Rubberduck.RegexAssistant
+{
+    public interface Atom : IDescribable
+    {
+        string Specifier { get; }
+    }
+
+    public class CharacterClass : Atom
+    {
+        public static readonly string Pattern = @"(?<!\\)\[(?<expression>.*?)(?<!\\)\]";
+        private static readonly Regex Matcher = new Regex("^" + Pattern + "$");
+
+        public bool InverseMatching { get; }
```

Comment:
Mine is closer to his existing one, and we use it in all (AFAIK) other places in RD.